packages/protocol/migrationsConfig.js

```diff
@@ -79,6 +79,7 @@ const DefaultConfig = {
       minor: 8,
       patch: 23,
     },
+    blockGasLimit: 8000000,
```

Comment:
Let's go with 20M, since we target half-full blocks via the GasPriceMinimum mechanism